crystal/ricr/shard.yml

```diff
@@ -0,0 +1,15 @@
+name: ricr
+version: 0.1.3
+
+authors:
+  - Julien Reichardt <contact@jrei.ch>
+
+targets:
+  server:
+    main: src/server.cr
+
+dependencies:
+  ricr:
+    github: ricr-web/ricr
```

Comment:
I forget

I prefer you add a constraint `here`
~~~yaml
version: ~> 0.1.3
~~~